aerogear/mobile-security-service
# feat(patternfly): adding intial patternfly4 structure and router

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8375

## What
- React Router
- Landing Page with Table, sortable and linkable by Row.

## How
Add an short answer for: How it was done? (E.g By removing this feature from ... OR By removing just the button but not its implementation ... ) 

## Verification Steps
Run the application as described in the [README](https://github.com/aerogear/mobile-security-service/tree/master/ui#mobile-security-service-ui)
Navigate to localhost:3000
You should see a table as below 
EDIT: Rows are currently not clickable due to refactor but this will be resolved in this JIRA https://issues.jboss.org/browse/AEROGEAR-8552
Rows are clickable however as there is only one route available at the moment, you will get the following error: 
`index.js:1446 Warning: You tried to redirect to the same route you're currently on: "/overview"`
This will be updated in this or a future pr.

<img width="958" alt="screenshot 2019-02-18 at 16 01 09" src="https://user-images.githubusercontent.com/6498727/52962963-7d313a80-3396-11e9-8c34-f5fbdce67ccb.png">


## Checklist:


## Additional Notes

This pulls together the initial structure using Patternfly 4 based on investigations done in https://github.com/aerogear/mobile-security-service/pull/33, https://github.com/laurafitzgerald/mobile-security-service/tree/pf4-table-poc and https://github.com/damienomurchu/mobile-security-service/tree/pf3
